src/app/project/CreateEditFeature.js

```diff
@@ -121,12 +121,24 @@ define([
         //      sent to toaster
         duplicateActionMsg: 'Can\'t add duplicate actions!',
 
+
         // Properties to be sent into constructor
 
+        // existingData: Object (optional)
+        // {
+        //     category: String,
+        //     retreatment: Boolean,
+        //     geometry: Geometry,
```

Comment:
how difficult would it be to only send the property if there are deltas? For instance, if they modify the geometry, pass the geometry. if they change retreatment, pass retreatment, same for actions. That would simplify the api from having to discover changes and do unnecessary work. I think it might be troublesome with point precision to check if the geometry has changed.